homework/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="./main.css">
+  <link href="https://fonts.googleapis.com/css?family=Play" rel="stylesheet">
+  <title>HYF Github Respositories</title>
+</head>
+
+<body>
+  <div id="container">
+    <div id="container-left" style="flex-grow: 1">
+      <header>
+        <img src="hyfrepo.png" alt="HYF Repositories">
+        <h1>HYF Github Repositories</h1>
+        <form action="/action_page.php">
```

Comment:
I suppose you copied this from somewhere? We're not doing php here! ;)

Reply:
sorry :)